satellite/metainfo/metainfo.go

```diff
@@ -129,11 +129,21 @@ func (endpoint *Endpoint) CreateSegment(ctx context.Context, req *pb.SegmentWrit
 		return nil, status.Errorf(codes.Unauthenticated, err.Error())
 	}
 
+	redundancy, err := eestream.NewRedundancyStrategyFromProto(req.GetRedundancy())
+	if err != nil {
+		return nil, err
+	}
+
+	maxPieceSize := eestream.CalcPieceSize(req.GetMaxEncryptedSegmentSize(), redundancy)
+
 	// TODO most probably needs more params
 	request := &pb.FindStorageNodesRequest{
 		Opts: &pb.OverlayOptions{
-			Amount:       int64(req.Redundancy.Total),
-			Restrictions: &pb.NodeRestrictions{},
+			Amount: int64(req.Redundancy.Total),
```

Comment:
Do you think we still need this TODO above?

Reply:
No idea. It was not me who put it :)
If you want, I can remove it.

Reply:
I mean, do you think we have all the necessary params? :)

Reply:
I think they are enough for now. I will remove the TODO comment.